test/binCases/config-type/function-promise/dist/entry.bundle.js

```diff
@@ -1 +1,19 @@
-!function(e){function r(n){if(t[n])return t[n].exports;var o=t[n]={i:n,l:!1,exports:{}};return e[n].call(o.exports,o,o.exports,r),o.l=!0,o.exports}var t={};r.m=e,r.c=t,r.d=function(e,t,n){r.o(e,t)||Object.defineProperty(e,t,{configurable:!1,enumerable:!0,get:n})},r.r=function(e){Object.defineProperty(e,"__esModule",{value:!0})},r.n=function(e){var t=e&&e.__esModule?function(){return e.default}:function(){return e};return r.d(t,"a",t),t},r.o=function(e,r){return Object.prototype.hasOwnProperty.call(e,r)},r.p="",r.w={},r(r.s=0)}([function(e,r){e.exports="entry.js"}]);
\ No newline at end of file
+!function(e) {
```

Comment:
This should be ignored from linters, right?

Reply:
Yes